doc/topics/installation/rhel.rst

```diff
@@ -149,7 +160,7 @@ Installing from ``epel-testing``
 ********************************
 
 When a new Salt release is packaged, it is first admitted into the
-``epel-testing`` repository, before being moved to the stable repo.
+``epel-testing`` repository, before being moved to the main stable repository.
```

Comment:
"main stable repository" is redundant, if anything this should be changed to "stable epel repository".